cypress/integration/e2e/application_spec.js

```diff
@@ -38,13 +38,28 @@ describe("Application Spec", () => {
 
     cy.wait("@applicationSave").then(() => {
         // move to drawer and check for update
-        cy.get("[data-cy='design-drawer']").click();
+        cy.get("[data-cy='application-drawer']").click();
         cy.wait(5000); // wait for seconds, because the subscritions cannot be tracked for now
         cy.get("#MUIDataTableBodyRow-applications-0 p").contains(bookInfoUrlUploadedName);
       })
   })
 
-  it("Deploy an Application", () => {
+  it("Validate an application", () => {
+    cy.get("[data-cy='application-drawer']").click();
+    cy.get("#MUIDataTableBodyRow-applications-0", {timeout: 30000})
+    cy.get("#MUIDataTableBodyRow-applications-0").click();
+    cy.get('[data-test-id="Search"]').type(bookInfoApp);
+    cy.intercept("/api/application*").as("applicationPost")
+    cy.wait("@applicationPost")
+    cy.get("body").then(body => {
+      if (body.find("[aria-describedby='notistack-snackbar'] #notistack-snackbar").length > 0) {
+        cy.get("[aria-describedby='notistack-snackbar'] #notistack-snackbar").should("not.contain", "Unable to render")
+      }
+    })
+    cy.get("#verify-design-btn").click();
```

Comment:
there should be assertion so that it can be known that validation was succesful or unsuccessful

Reply:
Please check.

Reply:
Waiting for the meshmodel/validate api to get in validate for MeshMap. To get t a better assertion (API call rather than just content assertion)
Will update it too.

Reply:
For now, I have used the "OK" text to check if validate modal is open or not.